+ '[' -z '' ']'
+ case "$-" in
+ __lmod_vx=x
+ '[' -n x ']'
+ set +x
Shell debugging temporarily silenced: export LMOD_SH_DBG_ON=1 for this output (/p/software/jurecadc/lmod/8.7.12/init/bash)
Shell debugging restarted
+ unset __lmod_vx
+ srun python ac1_01.py --beta=1 --time_intervals=64 --rolling=2
[0]PETSC ERROR: ------------------------------------------------------------------------
[0]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[0]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[0]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[0]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[0]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
srun: Job step aborted: Waiting up to 6 seconds for job step to finish.
[16]PETSC ERROR: ------------------------------------------------------------------------
[16]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[16]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[16]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[16]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[16]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[56]PETSC ERROR: ------------------------------------------------------------------------
[56]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[56]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[56]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[32]PETSC ERROR: ------------------------------------------------------------------------
[32]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[32]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 0
[8]PETSC ERROR: ------------------------------------------------------------------------
[8]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[8]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[8]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[8]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[8]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[12]PETSC ERROR: ------------------------------------------------------------------------
[12]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[12]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[12]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[12]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[12]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[4]PETSC ERROR: ------------------------------------------------------------------------
[4]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[4]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[4]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[4]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[4]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 4
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 8
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 12
p_Abort(r4): aborting on users request
p_Abort(r0): aborting on users request
p_Abort(r8): aborting on users request
p_Abort(r12): aborting on users request
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 16
[20]PETSC ERROR: ------------------------------------------------------------------------
[20]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[20]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[20]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[20]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[20]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 20
[48]PETSC ERROR: ------------------------------------------------------------------------
[48]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[48]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[48]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[48]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[48]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
p_Abort(r16): aborting on users request
[56]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[56]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[24]PETSC ERROR: ------------------------------------------------------------------------
[24]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[24]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[24]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[24]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[24]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 24
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 48
p_Abort(r20): aborting on users request
[52]PETSC ERROR: ------------------------------------------------------------------------
[52]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[52]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[52]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[52]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[52]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[28]PETSC ERROR: ------------------------------------------------------------------------
[28]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[28]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[28]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[28]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[28]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 28
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 56
p_Abort(r24): aborting on users request
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 52
p_Abort(r28): aborting on users request
[60]PETSC ERROR: ------------------------------------------------------------------------
[60]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[60]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[60]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[60]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[60]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 60
p_Abort(r48): aborting on users request
p_Abort(r56): aborting on users request
p_Abort(r60): aborting on users request
p_Abort(r52): aborting on users request
[32]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[32]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[32]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[36]PETSC ERROR: ------------------------------------------------------------------------
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 32
[36]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[36]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
p_Abort(r32): aborting on users request
[36]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[40]PETSC ERROR: ------------------------------------------------------------------------
[40]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[40]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[40]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[40]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[36]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
[40]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
[36]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 40
[44]PETSC ERROR: ------------------------------------------------------------------------
[44]PETSC ERROR: Caught signal number 15 Terminate: Some process (or the batch system) has told this process to end
[44]PETSC ERROR: Try option -start_in_debugger or -on_error_attach_debugger
[44]PETSC ERROR: or see https://petsc.org/release/faq/#valgrind and https://petsc.org/release/faq/
[44]PETSC ERROR: ---------------------  Stack Frames ------------------------------------
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 36
p_Abort(r40): aborting on users request
[44]PETSC ERROR: No error traceback is available, the problem could be in the main program. 
application called MPI_Abort(MPI_COMM_WORLD, 59) - process 44
p_Abort(r36): aborting on users request
p_Abort(r44): aborting on users request
srun: error: jrc0781: tasks 32-47: Terminated
srun: error: jrc0780: tasks 16-31: Terminated
srun: error: jrc0782: tasks 48-63: Terminated
srun: error: jrc0779: tasks 0-15: Terminated
+ touch ready
